cpp/rss_subset_aggregator.cc

```diff
@@ -0,0 +1,315 @@
+/** \file   rss_subset_aggregator.cc
+ *  \brief  Aggregates RSS feeds.
+ *  \author Dr. Johannes Ruscheinski (johannes.ruscheinski@uni-tuebingen.de)
+ *
+ *  \copyright 2021 Universitätsbibliothek Tübingen.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <algorithm>
+#include <map>
+#include <unordered_map>
+#include <unordered_set>
+#include <cinttypes>
+#include <cstring>
+#include "Compiler.h"
+#include "DbConnection.h"
+#include "DbResultSet.h"
+#include "EmailSender.h"
+#include "FileUtil.h"
+#include "HtmlUtil.h"
+#include "SqlUtil.h"
+#include "StringUtil.h"
+#include "SyndicationFormat.h"
+#include "Template.h"
+#include "UBTools.h"
+#include "util.h"
+#include "VuFind.h"
+#include "XmlWriter.h"
+
+
+namespace {
+
+
+[[noreturn]] void Usage() {
+    ::Usage("email_address [user_id]\n"
```

Comment:
1. if a rss feed is generated for a single user to show him in the frontend, why would we want to pass an email address here?
2. If the aggregation takes place for all users + email is sent to them, the email addresses from the vufind db should be used, so why would we want to pass an email address here? For cc? And do we want to put these mails into several cronjobs for each instance?